test/runtime-test.js

```diff
@@ -710,6 +710,41 @@ describe('BEMHTML compiler/Runtime', function() {
       test(function() {
       }, { tag: 'span', attrs: { a: 'b' } }, '<span a="b"></span>');
     });
+
+    it('should properly render undefined as attrs value', function () {
```

Comment:
@zxqfox I can do it, but I think it’s useless because of https://github.com/bem/bem-xjst/pull/167